src/server/routes/entity/entity.js

```diff
@@ -161,7 +161,8 @@ export function displayEntity(req: PassportRequest, res: $Response) {
 				markup,
 				page: entityName,
 				props: escapeProps(props),
-				script: '/js/entity/entity.js'
+				script: '/js/entity/entity.js',
+				title: `${getEntityLabel(props.entity)}: ${_.upperFirst(entityName)}`
```

Comment:
If you look in `getEntityLabel`, you'll see that in certain cases it will return an html fragment.
That's not going to work in the title, it'll look like "[object Object] Work" for example.

You could change `getEntityLabel` to add an optional boolean parameter `returnHTML` (defaulting to true) and return a string or html fragment according to that.

Reply:
I also would prefer the entity name to be in parentheses instead of the semicolon.
Like this "Aldous Huxley (Author)".

What do you think?

Reply:
I have created an extra boolean function for deleted entities. And also changed the title slightly as you asked for.